rfcs/text/pe/pe.yml

```diff
@@ -0,0 +1,257 @@
+
+        - name: entropy
+          level: extended
+          description: Measurement of entropy randomness in the resources section.
+          type: long
+          example: 0, 1
+
+        - name: sha256
+          level: extended
+          description: SHA256 hash of resources section.
+          type: keyword
+          example: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+
+        - name: language
+          level: extended
+          description: Language identification.
+          type: keyword
+          example: "CHINESE SIMPLIFIED"
+
+        - name: types
+          level: extended
+          type: keyword
+          short: List of resource types.
+          description: >
+            Digest of resource types. Key is resource type (as string) and value
+            is how many resources there are of that specific type (as integer)
+          example: { "RT_VERSION": 1, "RT_MANIFEST": 1 }
+
+    - name: exports
+      level: extended
+      type: keyword
+      description: >
+        List of symbols exported by PE
+      example: DllInstall, DllRegisterServer, DllUnregisterServer
+
+    - name: creation_date
+      level: extended
+      short: Build or compile date.
+      description: >
+        Extracted when possible from the file's metadata. Indicates when it was
+        built or compiled. It can also be faked by malware creators.
+      type: date
+      example: "2020-11-05T17:25:47.000Z"
+
+    - name: authentihash
+      level: extended
+      description: >
+        Authentihash of the PE file.
+      type: keyword
+      example: ac9555d914bbb112ecc5f15bb9887ca8371f493ab0941344e976bb8410c8aa78
+
+    - name: compile_timestamp
+      level: extended
+      description: >
+        Compile timestamp of the PE file.
+      type: date
+      example: "2020-11-05T17:25:47.000Z"
+
+    - name: compiler
+      level: extended
+      description: >
+        Compiler information for the PE file.
+      type: object
+      fields:
+        - name: name
+          level: extended
+          type: keyword
+          description: >
+            Name of the compiler
+          example: Clang
+
+        - name: version
+          level: extended
+          type: keyword
+          description: >
+            Version of the compiler.
+          example: 11.0.0
+
+    - name: rich_header
+      level: extended
+      description: >
+        Rich Header information for the PE file.
+      type: object
+      fields:
+        - name: hash
+          level: extended
+          description: >
+            Hash of the header for the PE file
+          type: object
+          fields:
+            - name: md5
+              level: extended
+              type: keyword
+              description: >
+                MD5 hash of the header for the PE file.
+              example: 5aa1aa0f2b4be70397a1e9e2b87627cd
+
+    - name: entry_point
+      level: extended
+      description: >
+        Entry point of the PE file.
```

Comment:
I think it's worth adding that this is the relative byte offset to the base of the file/image.

Reply:
Updated.